sockshop/catalogue/dump.sql

```diff
@@ -0,0 +1,79 @@
+CREATE USER IF NOT EXISTS 'catalogue_user' IDENTIFIED BY 'default_password';
```

Comment:
yeah, was undecided how/where we should structure this. Probably makes sense to have it in the service with then scripts to copy in to the proper location?